content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,23 +1,177 @@
+
+Once the product variants feature is activated, product attributes to apply on variants can be
+created and edited, either from the :menuselection:`Inventory app --> Configuration --> Settings`
+by clicking the :guilabel:`internal link arrow` next to :guilabel:`Attributes`, or by clicking
+:menuselection:`Inventory app --> Configuration --> Attributes`.
+
+Once on the :guilabel:`Attributes` page, either click into an existing attribute, or click
+:guilabel:`Create` to create a new one. Doing so reveals a new, blank form for customizing an
+attribute. Click :guilabel:`Edit` to make changes.
+
+Assign an :guilabel:`Attribute Name`, and click the drop-down menu next to the :guilabel:`Category`
+field to assign this attribute to a category. Then, select the desired options next to the
+:guilabel:`Display Type` and :guilabel:`Variants Creation Mode` fields. Once the desired options are
+selected, click :guilabel:`Add a line` under the :guilabel:`Attribute Values` tab to add a new
+value.
+
+.. tip::
+   At the far right of the :guilabel:`Value` row, there is a :guilabel:`Is custom value` checkbox.
+   If selected, this value will be recognized as a custom value, which allows customers to type
+   special customization requests upon ordering a custom variant of a product.
+
+.. example::
+   .. image:: product_variants/product-variants-attribute.png
+      :align: center
+      :alt: Product variant attribute configuration screen.
+
+Once all desired :guilabel:`Values` have been added, click :guilabel:`Save` to save the new
+attribute.
+
+Add product variants on the product form
+========================================
+
+Created attributes can be applied on specific variants for particular products. To add product
+variants to a product, navigate to the product form by going to :menuselection:`Inventory app -->
+Products --> Products`. To make changes to the product, click :guilabel:`Edit`. Then, click the
+:guilabel:`Variants` tab.
+
+Under the :guilabel:`Attribute` header, click :guilabel:`Add a line` to add a new attribute, and
+select one to add from the drop-down menu. Then, under the :guilabel:`Values` header, click the
+drop-down menu to reveal the list of possible values created previously. Add all the desired values
+by clicking on them. Repeat this process for any additional attributes that should be added to the
+product. Once finished, click :guilabel:`Save` to save changes.
+
+.. image:: product_variants/product-variants-product-form.png
+   :align: center
+   :alt: Product form variants tab with values and attributes.
+
+.. tip::
+   :abbr:`BoM (bill of materials)` products with multiple variants that are manufactured in-house
+   should either have a 0,0 reordering rule set up, or have their replenishment routes set to
+   *Replenish on Order (MTO)*.
+
+Apply BoM components to product variants
+========================================
+
+Next, create a new :abbr:`BoM (bill of materials)`, or edit an existing one, by going to
```

Comment:
Because I like to show how to do things from scratch, for people actually starting from scratch. I think it's helpful to include all instructions from beginning to end even if it's a little long-winded